src/Maestro/Maestro.Web/.config/settings.Development.json

```diff
@@ -3,5 +3,6 @@
   "AppConfigurationUri": "https://maestrolocal.azconfig.io/",
   "BuildAssetRegistry": {
     "ConnectionString": "Data Source=localhost\\SQLEXPRESS;Initial Catalog=BuildAssetRegistry;Integrated Security=true"
-  }
+  },
+  "ForceLocalApi": true
```

Comment:
This will break the local dev scenario where `ng serve` + `dotnet run` are combined to debug the angular bits.

Reply:
Oh crap, meant to leave this false. :-)